pkgs/tools/system/nvtop/default.nix

```diff
@@ -24,13 +24,13 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "nvtop" + pname-suffix;
-  version = "2.0.2";
+  version = "2.0.3";
 
   src = fetchFromGitHub {
     owner = "Syllo";
     repo = "nvtop";
-    rev = version;
-    sha256 = "sha256-TlhCU7PydzHG/YMlk922mxEJ3CZw40784U0w1YawI3I=";
+    rev = "59975536974039fab398d0811d168758f523286a";
```

Comment:
can you please pick the version and add the commit via fetchpatch as patche.

Reply:
:ok_hand: